packages/kit/src/views/Swap/pages/components/SwapInputActions.tsx

```
@@ -81,21 +80,26 @@ const SwapInputActions = ({
         {showPercentageInput ? (
           <>
             {needSwapPercentageInputStage.map((stage) => (
-              <Button
-                height="$5"
+              <Badge
                 key={`swap-percentage-input-stage-${stage}`}
-                size="small"
+                role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)